apps/src/gamelab/blocks.js

```diff
@@ -154,7 +153,7 @@ const customInputTypes = {
   costumePicker: {
     addInput(blockly, block, inputConfig, currentInputRow) {
       let buttons;
-      if (experiments.isEnabled('sprite-costumes')) {
+      if (getStore().getState().pageConstants.showAnimationMode) {
```

Comment:
Limit to just Spritelab?

Reply:
As opposed to Gamelab? This should already be only Spritelab because it is blockly blocks.